roles/change-paths/tasks/main.yml

```diff
@@ -0,0 +1,167 @@
+---
+# Ansible role for changing paths after a site has been uploaded.
+# https://github.com/SU-SWS/ansible-playbooks
+# ==================================================================
+#
+# This role runs a number of tasks to transform the database in such a way that
+# images and absolute paths will work in the new ACSF environment. If any part
+# of this role fails for a site, restart from the tag "download-site".
+#
+# INPUTS:
+#   acsf_site_name
+#   inventory_hostname
+#   sitefactory_environment
+#   drush_environment
+#   server_alias
+#   site_prefix
+#   stack
+#   server
+#   dept_site
+#   wait_time
+#
+# OUTPUTS:
+#   file_public_path
+#
+# ALTERNATIVE ROLES:
+# --
+#
+# REQUIREMENTS:
+# --
+#
+# KNOWN ISSUES:
+# --
+
+##############################################################################
+# Database surgery: Find and replace anything that the user may have entered #
+# that references the old site.                                              #
+# See https://stanfordits.atlassian.net/browse/SITES-248 for details.        #
+##############################################################################
+
+- name: Set sites_service variable
```

Comment:
I would add a comment here like `# Example output: "people"`

Reply:
Changed in cleanup-refactor

Reply:
Yeah, I have a bunch of changes coming to `cleanup-refactor`

Reply:
Added: https://github.com/SU-SWS/ansible-playbooks/blob/79c6fb5acddbe945fc988194b165d694384a5bd7/roles/change-paths/tasks/main.yml#L49